### PR TITLE
Developer images

### PR DESCRIPTION
**Pull request by Djelibeybi**
This adds several new Oracle Linux based language images for use as base images for development or language specific purposes.

### CODE REVIEW

**Comment by Djelibeybi** [comment]
@gvenzl @mriccell I need one/both of you to review/approve this but please don't merge. I want to squash the commits prior to merge.

File: OracleLinuxDevelopers/oraclelinux7/nodejs/10-oracledb/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+LABEL "provider"="Oracle"                                               \
+      "issues"="https://github.com/oracle/docker-images/issues"
+
+RUN yum -y install oracle-nodejs-release-el7 oracle-release-el7 && \
+    yum-config-manager --disable ol7_UEKR5 && \
+    yum-config-manager --disable ol7_developer_nodejs12 && \
+    yum-config-manager --enable ol7_developer_nodejs10 && \
+    yum -y install nodejs node-oracledb-node10 && \
+    rm -rf /var/cache/yum/*
+
```
Comment (cjbj) [inline OracleLinuxDevelopers/oraclelinux7/nodejs/10-oracledb/Dockerfile:14]: Add `ENV NODE_PATH=/usr/lib/node_modules`

Comment (Djelibeybi) [reply OracleLinuxDevelopers/oraclelinux7/nodejs/10-oracledb/Dockerfile:14]: Done.

Comment (cjbj) [reply OracleLinuxDevelopers/oraclelinux7/nodejs/10-oracledb/Dockerfile:14]: Thanks,

**Comment by Djelibeybi** [comment]
@gvenzl can you please re-approve this PR? Your approval was flagged as stale when I added the `NODE_MODULES` variable for @cjbj

**Pull request merged**